### PR TITLE
Word object model

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Patterned on the new Excel object model article as per plan.

We will add a graphic like the one in the Excel article in a later PR.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a23f0e2](https://github.com/OfficeDev/office-js-docs-pr/commits/a23f0e29b7a2dda900156e33a2aa01c2de0fa506)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/overview/word-add-ins-reference-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/reference/overview/word-add-ins-reference-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/word/word-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/word/word-add-ins-core-concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-core-concepts?branch=pr-en-us-2157) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C4%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009042127028771-2157%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=si9TVdzmtniRF9%2FlvJFs9rkzkUi54xfolzu45lX3C2o%3D&st=2020-09-04T21%3A22%3A41Z&se=2020-10-05T21%3A27%3A41Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[64032fa](https://github.com/OfficeDev/office-js-docs-pr/commits/64032faef37fb665754641f3e1220f06a46b4b40)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/overview/word-add-ins-reference-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/reference/overview/word-add-ins-reference-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/word/word-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/word/word-add-ins-core-concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-core-concepts?branch=pr-en-us-2157) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C4%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009042128209362-2157%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=lpHycdK6VuUl6Z5nG30M%2FqVKCy4myjYvkwKUphgwhBI%3D&st=2020-09-04T21%3A24%3A00Z&se=2020-10-05T21%3A29%3A00Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[066d453](https://github.com/OfficeDev/office-js-docs-pr/commits/066d453185e782ad70c283b50b684fd7f750cf13)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/overview/word-add-ins-reference-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/reference/overview/word-add-ins-reference-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/word/word-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/word/word-add-ins-core-concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-core-concepts?branch=pr-en-us-2157) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C4%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009042131038049-2157%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=F1%2BJLVZSubmfkAC39e%2FetnmFh4JP61A5ibLEGMUWbU4%3D&st=2020-09-04T21%3A26%3A44Z&se=2020-10-05T21%3A31%3A44Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3dcb4a8](https://github.com/OfficeDev/office-js-docs-pr/commits/3dcb4a8b6689c73fff969ca1970d463bbef016f2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-js-api-common-api.png](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/excel/excel-js-api-common-api.png) | :white_check_mark:Succeeded | |
[docs/reference/overview/word-add-ins-reference-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/reference/overview/word-add-ins-reference-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/word/word-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/word/word-add-ins-core-concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-core-concepts?branch=pr-en-us-2157) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009080139014726-2157%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=FlxNYCdNBx%2B6lierim29OIezLe4Oi7SuodvPfypXp1c%3D&st=2020-09-08T01%3A34%3A37Z&se=2020-10-09T01%3A39%3A37Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by alison-mk — APPROVED** [review]
Thanks for doing this, Ricky! Looks great. Added a couple questions + suggestions.

File: docs/word/word-add-ins-core-concepts.md [hunk 1]
```diff
@@ -1,67 +1,44 @@
...
 
-# Fundamental programming concepts with the Word JavaScript API
+# Word JavaScript object model in Office Add-ins
 
-This article describes concepts that are fundamental to using the [Word JavaScript API](../reference/overview/word-add-ins-reference-overview.md) to build add-ins for Word 2016 or later.
+This article describes concepts that are fundamental to using the [Word JavaScript API](../reference/overview/word-add-ins-reference-overview.md) to build add-ins. It introduces core concepts that are fundamental to using the API.
 
-## Referencing Office.js
+> [!IMPORTANT]
+> See [Using the application-specific API model](../develop/application-specific-api-model.md) to learn about the asynchronous nature of the Word APIs and how they work with the workbook.
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/word/word-add-ins-core-concepts.md:13]: Good catch that was left over from a copy paste. Will fix.

File: docs/word/word-add-ins-core-concepts.md [hunk 2]
```diff
@@ -1,67 +1,44 @@
...
-    // Use the context argument to get access to the Word document.
-    Word.run(function (context) {
-
-        // Create a proxy object for the document.
-        var thisDocument = context.document;
-        // ...
-    })
-})();
-```
+* The **Document** contains the **Section**s, and document level entities such as settings and custom XML parts.
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/word/word-add-ins-core-concepts.md:32]: Good catch.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[364afda](https://github.com/OfficeDev/office-js-docs-pr/commits/364afda87efab3ac8b96a94814e4551f76030e52)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-js-api-common-api.png](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/excel/excel-js-api-common-api.png) | :white_check_mark:Succeeded | |
[docs/reference/overview/word-add-ins-reference-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/reference/overview/word-add-ins-reference-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/word/word-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/word/word-add-ins-core-concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-core-concepts?branch=pr-en-us-2157) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009082005456621-2157%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=IDnR0EcKPGmKRdmWtsARMnZA3gGJn4iuQ7V4ccDk0Kc%3D&st=2020-09-08T20%3A01%3A20Z&se=2020-10-09T20%3A06%3A20Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[da3a1fd](https://github.com/OfficeDev/office-js-docs-pr/commits/da3a1fdbea5a35cb73052b6b9dc40e9966f4aaa0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-js-api-common-api.png](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/excel/excel-js-api-common-api.png) | :white_check_mark:Succeeded | |
[docs/reference/overview/word-add-ins-reference-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/reference/overview/word-add-ins-reference-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?branch=pr-en-us-2157) |
[docs/word/word-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/word-object-model/docs/word/word-add-ins-core-concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-core-concepts?branch=pr-en-us-2157) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009082018405336-2157%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=os78%2F7OSE0NQK1EYmEcrW8UMetWTbbUFq%2FPX5d6LF6E%3D&st=2020-09-08T20%3A14%3A16Z&se=2020-10-09T20%3A19%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**